testing-guidance/workflow-tests/track-view/track-view-tests.md

```diff
@@ -0,0 +1,88 @@
+## **Track View Workflow Tests**
+
+Testing in this area should focus on the functionality of the Track View. This is a general tool for the creating, editing and exporting of cutscenes.
+
+## General Docs
+
+* https://www.o3de.org/docs/user-guide/visualization/cinematics/track-view/
+##  Common Issues to Watch For
+
+Test guidance will sometimes note specific issues to watch for. The common issues below should be watched for through all testing, even if unrelated to the current workflow being tested.
+
+* The Camera parameter does not take effect.
+* Track View keyframe attributes cannot be modified.
+* Nodes cannot be added to the Sequence.
+* Unable to switch between multiple cameras.
+* Editor or Track View crashes.
+* Track View does not display the recorded Sequence.
+
+# Workflows
+
+## Area: Track View LKG Workflow
+
+**Project Requirements**
+
+* You've read through the Using the Track View Editor docs.
+* FFMpeg plugin installed.
+
+**Editor Platforms:**
+
+* Windows
+* Linux
+
+**Game Launcher Supported Platforms:**
+
+* Windows
+* Linux
+
+**Product:** Visible and responsive Track View editor.
+
+**Suggested Time Box:** 15 minutes per platform.
+
+| Workflow                                        | Requests           | Things to Watch For |
+|-------------------------------------------------|--------------------|---------------------|
+|Track View editor can be launched and configured| <ol><li>Open Track View editor by multiple ways including:</li> <ul><li>Tools menu</li><li>Keyboard shortcut ( a shortcut can be assigned via Edit > Editor Settings > Keyboard Customization > Customize Keyboard)</li></ul> <li>Perform resize, maximize, minimize undock, dock actions with Track View editor and its panes and drag them around the Editor. </li><li> Close and reopen the Track View editor.</ol> | <ul><li>Track View editor launch method does not work or does not launch quickly.</li> <li>Track View editor not retaining configuration settings on close/relaunch. <li>Track View editor not retaining configuration settings on close/relaunch.</li><li>Track View panes not rendering, having rendering issues after being altered.</li><li>Not being able to resize, dock or drag panes borders.</li></li></ul>  |
+|A Sequence can be created and configured                | <ol> <li>Add Sequence to the Track View by multiple ways including:</li> <ul><li>Tools menu</li> <li>Node shortcut</li> </ul> <li>Name the Sequence.</li> <li>Edit Sequence properties by multiple ways including:</li> <ul> <li>Enabling autostart</li> <li>Editing Sequences name</li> <li>Changing start/end timings.</li> <li>Changing display time from seconds to frames.</li> <li>Selecting different Out of Range options.</li> </ul> <li>Delete and recreate the Sequence.</li> </ol> | <ul><li>The Sequence name is left empty.</li><li>Created Sequences do not appear in the Sequence dropdown.</li><li>Sequence properties changes do not take place in the Track Editor and the Curve Editor.</li></ul>  |
```

Comment:
update: Please review how formatting looks in markdown and be consistent with numbers (instead of bullets) - when looking at the markdown preview the formatting looks a bit off in places.